frontends/web/src/routes/device/bitbox02/bitbox02.tsx

```diff
@@ -273,9 +259,9 @@ class BitBox02 extends Component<Props, State> {
       settingPassword: true,
       createWalletStatus: 'setPassword',
     });
-    apiPost(this.apiPrefix() + '/set-password').then(({ code, success }) => {
-      if (!success) {
-        if (code === errUserAbort) {
+    setPassword(this.props.deviceID).then((response) => {
+      if (!response.success) {
+        if (response.code === errUserAbort) {
```

Comment:
same, can't destruct argument anymore as code is optional (only in case of success = false).